components/shared/Avatar.vue

```diff
@@ -21,4 +34,14 @@ withDefaults(
     size: 64,
   },
 )
+
+const evmAvatarSvg = computed(() =>
+  toSvg(props.value, props.size - 2, {
+    padding: 0.1,
+  }),
+)
```

Comment:
why?

Reply:
1. `padding: 0.1` would make it look better
2.  add `props.size - 2`  because the outer border will cause the image to shift